src/Components/Field/index.js

```diff
@@ -0,0 +1,36 @@
+import Field from './Field'
+import { connect } from 'react-redux'
+import { plantSeeds, SEEDS_TYPES, harvest } from './../../Redux/State/Farm'
+
+//findSeed :: (String, [SeedType]) -> SeedType
+const findSeed = (seedName, seedTypes)=>
+    seedTypes.find(seed => seed.name === seedName)
+
+
+// mapStateToProps :: State -> Props
+const mapStateToProps = (state, ownProps) => ({
+    seedTypes : SEEDS_TYPES,
```

Comment:
Plus j'y pense, plus ça me pose problème que ce seedtypes soit injecté de cette façon là :thinking: 

A mon avis on ne devrais pas l'importer du module ```Farm```, mais ca devrais être un module a part entière. Comme on sait que tu va probablement créer un système de marché plus tard, je pense que le mieux serait de créer un module `Marketplace` dans `/Redux/State`. Comme ca, tu pourra l'injecter ici directement en faisant

```seedTypes: state.Marketplace.seedTypes```

et ca sera beaucoup plus simple a faire évoluer et a tester :wink: 

Qu'en penses tu ? Peux tu faire cette petite refacto ? :angel:

Reply:
J'espèèère que j'ai fais ce qu'il fallait xD ! We will see...